client/layout/index.jsx

```diff
@@ -320,6 +321,13 @@ class Layout extends Component {
 						placeholder={ null }
 						id="notices"
 					/>
+					{ config.isEnabled( 'layout/launchpad-banner' ) && (
+						<MainBanner
```

Comment:
I'm assuming we're intentionally not async loading this to avoid a layout shift when it loads? In any case, this will be loaded for everyone, so let's keep an eye on bundle size.